clients/src/main/java/org/apache/kafka/clients/producer/ProducerConfig.java

```diff
@@ -300,7 +300,7 @@
                                 .define(RETRIES_CONFIG, Type.INT, Integer.MAX_VALUE, between(0, Integer.MAX_VALUE), Importance.HIGH, RETRIES_DOC)
                                 .define(ACKS_CONFIG,
                                         Type.STRING,
-                                        "1",
+                                        "all",
                                         in("all", "-1", "0", "1"),
                                         Importance.HIGH,
```

Comment:
@d8tltanc in your PR with the test adjustments, please reduce the priority of this. Either `LOW` or `MEDIUM`, we can discuss more in the PR.